Overview.bs

```diff
@@ -2042,6 +2048,17 @@ way:
 4. Lastly, use glyph keyed patches to update the remaining tables, here much smaller fine-grained segments can be utilized without
     requiring too many patches.
 
+The mixed patch type encoding is expected to be the more commonly used approach since many fonts will fall somewhere in between the two
```

Comment:
The more I think about the construction "the more commonly" the less feeling I have of what level of emphasis I'm supposed to understand from it. We could say " the most commonly" and that would be clear, or we could say "a more commonly" which is vague but self-consistently vague. 

The description "many fonts" seems like it captures the desired (?) vagueness on its own, however, so how about just "... is expected to be a commonly used approach ..."

Reply:
Updated to "is expected to be a commonly used approach"